tasks.py

```diff
@@ -1 +1,12 @@
+from invoke import task
+
 from dmdevtools.invoke_tasks import library_tasks as ns
+
+
+@task(ns["virtualenv"], ns["requirements_dev"])
+def test_mypy(c):
+    c.run("mypy dmutils/")
```

Comment:
```suggestion
    c.run("mypy dmapiclient/")
```
I _think_ this is the only reason why the checks are failing - mypy seems happy on my machine

Reply:
Might be worth instead adding a `mypy.ini` file

```
[mypy]
files = dmapiclient
```

Then running `mypy` should just do the right thing.
```

Reply:
Huh. I didn't know about that. We already have a `mypy.ini`, so that sounds most useful

Reply:
Fixed and rebased against main